bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/AndroidDebugBridgeDevice.java

```diff
@@ -374,6 +379,220 @@ public void powerOffDevice()
         }
     }
 
+    public void startIntent(String command) throws NumberFormatException, AndroidDebugBridgeDeviceException,
```

Comment:
Take care, the method `handleCommandInternal` could then throw NumberFormatException which is not caught by the method handleCommand